python/semantic_kernel/agents/kernel_agent.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from semantic_kernel.agents.agent import Agent
+from semantic_kernel.utils.experimental_decorator import experimental_class
+
+
+@experimental_class
+class KernelAgent(Agent):
```

Comment:
this doesn't add much and it's optional, could we skip this class, or is this laying the groundwork for other types?

Reply:
Laying groundwork for other types.